src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -30,36 +48,153 @@ const defaultStylingConfig = {
   height: 400,
   maxWidth: 720,
   maxHeight: 530,
+  pointSize: 5,
+  pointOpa: 5,
+  pointStyle: 'circle',
+  toggleInvert: '#FFFFFF',
 };
 
+const { Panel } = Collapse;
+
 const DataIntegration = () => {
-  const [plotConfig, setPlotConfig] = useState(defaultStylingConfig);
+  const router = useRouter();
+  const { experimentId } = router.query;
+
+  const [selectedPlot, setSelectedPlot] = useState('sample');
```

Comment:
The functionality for the different plots and switching between each of them is actually part of a different ticket (in [this PR](https://github.com/biomage-ltd/ui/pull/96)), the addition of something like this here too might create some unnecessary merge conflicts.

Reply:
I see I see. I was using it to test for the if statements to check if conditionals for the different parts of the styling (if they show up properly or not). I think that justifies the code to be there.